### PR TITLE
Add option to show error when user does /home with no home set.

### PR DESCRIPTION
**Pull request by ghost**
Title mostly says it all. The pre-existing behaviour for when a user runs /home and has no Essentials home or bed home is to just call the PlayerRespawnEvent. This, in my mind, not only adds confusion to whether or not the player has a home set, but also adds compatibility with plugins that listen for the PlayerRespawnEvent and change the teleport location (in my case, players were being sent to random locations when /home was ran due to my random spawn plugin).

This is my first PR, so excuse if I did something wrong here. Thanks for taking the time, and thanks for dedicating your time to such a great plugin :)

### CODE REVIEW

**Review by Ichbinjoe — APPROVED** [review]

**Review by pop4959 — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

File: Essentials/src/com/earth2me/essentials/Settings.java [hunk 1]
```diff
@@ -56,6 +56,12 @@ public Settings(IEssentials ess) {
         reloadConfig();
     }
 
+    @Override
+    public boolean errorWhenNoHome() {
+        return config.getBoolean("error-when-no-home", false);
+
```
Comment (Ichbinjoe) [inline Essentials/src/com/earth2me/essentials/Settings.java:62]: Extra line

File: Essentials/src/net/ess3/api/ISettings.java [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 
 public interface ISettings extends com.earth2me.essentials.ISettings {
-}
+}
```
Comment (Ichbinjoe) [inline Essentials/src/net/ess3/api/ISettings.java:5]: Remove file from diff

File: Essentials/src/messages_en.properties [hunk 3]
```diff
@@ -326,6 +326,7 @@ north=N
 northWest=NW
 noGodWorldWarning=§4Warning\! God mode in this world disabled.
 noHomeSetPlayer=§6Player has not set a home.
+noHomeSet=§4You have no home set.
```
Comment (Ichbinjoe) [inline Essentials/src/messages_en.properties:329]: This will be overwritten by crowdin, remove

**Review by Ichbinjoe — CHANGES_REQUESTED** [review]
Some minor formatting things

File: Essentials/src/messages_en.properties (hunk 3 above)
Comment (ghost) [reply Essentials/src/messages_en.properties:329]: How would I go about adding a translatable message then? Sorry, new to this 😛

Comment (triagonal) [reply Essentials/src/messages_en.properties:329]: The only language file you should edit is `messages.properties` with the default English translation. All other locale files are automatically updated by Crowdin after the main properties file is edited.

Comment (ghost) [reply Essentials/src/messages_en.properties:329]: Okay, thank you.

**Comment by mdcfe** [comment]
#2757 implements this feature and has already been merged, so this PR is no longer necessary. Nevertheless, thanks for contributing your time, and apologies for the confusion and extra work you put in.

**Pull request closed without merging**